Documentation/data-collection.md

```diff
@@ -712,6 +712,42 @@ data:
     #
     # imageregistry:operations_count:sum is the total number of image pushes and pulls executed in the internal registry.
     - '{__name__="imageregistry:operations_count:sum"}'
+    #
+    # owners: (@openshift/team-logging)
+    #
+    # log_logging_info  gives cluster-logging-operator version, managedStatus, healthStatus specific info.
```

Comment:
`log_logging_info  gives`
should remove one space after `log_logging_info`
`log_logging_info gives`

Reply:
@pmoogi-redhat sorry, we have our rules to review the PR, accept the PR means we don't have concern for the PR, and it is easy to fix the issue